File: tests/fixtures/code/sample.c
// Greets someone by name.
#include <stdio.h>

void greet(const char *name) {
    printf("Hello, %s!\n", name);
}

int main(void) {
    const char *friends[] = {"Ada", "Grace", "Margaret"};
    for (int i = 0; i < 3; i++) {
        greet(friends[i]);
    }
    return 0;
}

File: tests/fixtures/code/sample.rs
// Greets someone by name.
fn greet(name: &str) -> String {
    let message = format!("Hello, {}!", name);
    return message;
}

fn main() {
    let friends = ["Ada", "Grace", "Margaret"];
    for friend in friends.iter() {
        println!("{}", greet(friend));
    }
}
